modules/core/shared/src/main/scala/coursier/core/Reconciliation.scala

```diff
@@ -13,66 +13,52 @@ sealed abstract class Reconciliation {
-  private[coursier] final val LatestRelease = "latest.release"
-  private[coursier] final val LatestStable = "latest.stable"
+  private final val LatestIntegration = "latest.integration"
+  private final val LatestRelease = "latest.release"
+  private final val LatestStable = "latest.stable"
 
-  abstract class AbstractReconciliation extends Reconciliation {
-    protected def mergeVersionConstraints(constraints: Seq[VersionConstraint]): Option[VersionConstraint]
-
-    protected def splitStandard(versions: Seq[String]): (Seq[String], Seq[String]) =
-      versions.distinct.partition {
-        case LatestIntegration => false
-        case LatestRelease     => false
-        case LatestStable      => false
-        case _                 => true
-      }
-
-    protected def retainLatestOpt(latests: Seq[String]): Option[String] =
-      if (latests.isEmpty) None
-      else if (latests.lengthCompare(1) == 0) latests.headOption
-      else {
-        val set = latests.toSet
-        val retained =
-          if (set(LatestIntegration))
-            LatestIntegration
-          else if (set(LatestRelease))
-            LatestRelease
-          else {
-            // at least two distinct latest.* means we shouldn't even reach this else block anyway
-            assert(set(LatestStable))
-            LatestStable
-          }
-        Some(retained)
-      }
+  private def splitStandard(versions: Seq[String]): (Seq[String], Seq[String]) =
+    versions.distinct.partition {
+      case LatestIntegration => false
+      case LatestRelease     => false
+      case LatestStable      => false
+      case _                 => true
+    }
 
-    protected def retainStandardOpt(standard: Seq[String]): Option[String] =
-      if (standard.isEmpty) None
-      else if (standard.lengthCompare(1) == 0) standard.headOption
-      else {
-        val parsedConstraints = standard.map(Parse.versionConstraint)
-        mergeVersionConstraints(parsedConstraints)
-          .flatMap(_.repr)
-      }
-  }
+  private def retainLatestOpt(latests: Seq[String]): Option[String] =
+    if (latests.isEmpty) None
+    else if (latests.lengthCompare(1) == 0) latests.headOption
+    else {
+      val set = latests.toSet
+      val retained =
+        if (set(LatestIntegration))
+          LatestIntegration
+        else if (set(LatestRelease))
+          LatestRelease
+        else {
+          // at least two distinct latest.* means we shouldn't even reach this else block anyway
+          assert(set(LatestStable))
+          LatestStable
+        }
+      Some(retained)
+    }
 
-  /**
-   * Implements the basic reconciliation rule based on `VersionConstraint.merge`.
-   */
-  case object Basic extends AbstractReconciliation {
-    /**
-     * Reconcile multiple version candidate.
-     *
-     * Returns `None` in case of conflict.
-     */
-    override def apply(versions: Seq[String]): Option[String] = {
+  case object Default extends Reconciliation {
```

Comment:
I avoided the term Default because default also means fallback, i.e. you could create a setting to use Relaxed _by default_, and Default/Basic for some modules X.

Reply:
That sounded ok to me… (does it really feel that confusing?)

`Default` is the default in coursier if users don't change anything. If they ask for `Relaxed` to be used instead, they aren't using the default anymore.